code/edge_detection.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>I'd say lets iterate on the previously used algorithms and get a better understanding and results out of them, before we get deeper into Fourier transformations. But great imitative!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>